highlight.io/components/Competitors/competitors.ts

```diff
@@ -44,4 +45,5 @@ export const COMPETITORS: { [k: string]: Competitor } = {
 	'highlight-vs-inspectlet': InspectletSpec,
 	'highlight-vs-datadog': DatadogSpec,
 	'highlight-vs-sentry': SentrySpec,
+	'highlight-vs-site24x7': Site24x7Spec,
```

Comment:
could we also update the our-competitors page to have all of these in there?

https://highlight-landing-git-site24x7-competitor-page-highlight-run.vercel.app/docs/general/company/our-competitors